frontend/src/components/dashboard/DraggableWrapper.js

```diff
@@ -290,6 +292,20 @@ export class DraggableWrapper extends Component {
     );
   };
 
+  renderLogo = () => {
+    const { indicators, indicatorsUpdated } = this.state;
+
+    return indicators.length > 0 || !indicatorsUpdated ? (
```

Comment:
IMHO it's fine to show the logo until the indicators are loaded.
It would look better than having a blank screen until we get something.
wdyt?

Reply:
Frankly looks weird to see it for a splash second. Gives the impression to the user that something not loaded properly or something is wrong.

Reply:
@petrican  makes sense. pls just rename the var to `indicatorsLoaded` because that's the actual meaning your are checking.